main.cpp

```diff
@@ -134,6 +136,22 @@ int Synthesizer::Keyboard::getNumOfKeysPressed(int numOfMidiEvents)
     return numOfMidiEvents;
 }
 
+void Synthesizer::Keyboard::changeVelocity(int midiValue)
+{
+    int velocityValue = 0;
+    bool midiTriggered = true;
+    int velocityChanged = midiTriggered + velocityValue + midiValue;
+
+    while(midiTriggered)
+    {
+        ++velocityValue;
+        ++velocityChanged;
+        std::cout << "New velocity" << std::endl;
+        if(velocityChanged >= 1)
+        break;
```

Comment:
Indent the body of any `if` blocks, even if they're a single statement/single line
```
if( condition )
    doSomething();
```
not:
```
if( condition )
doSomething();
```
- [ ] revise